### PR TITLE
code generator snippets

### PR DESCRIPTION
**Pull request by trejdych**
Hello Robert,

I've added snippets for the new code generate syntax.

I think for the new syntax is more important to cover keep alive/autodispose than family.

I'll update readme when you accept snippets :)

sample gifs
CleanShot 2022-10-08 at 10 18 27
CleanShot 2022-10-08 at 10 29 29

### CODE REVIEW

**Comment by pythonhubdev** [comment]
Looks great

**Comment by RobertBrunhage** [comment]
Looks good to me!

Feel free to update readme and the version!

**Comment by trejdych** [comment]
Updated :)

**Review by RobertBrunhage — CHANGES_REQUESTED** [review]
One more small note:

Consider adding the snippets to the README as well!

**Comment by trejdych** [comment]
Sorry, I don't understand. I did it in the last commit. Am I missing something?

**Comment by trejdych** [comment]
ahh you mean CHANGELOG?

**Comment by trejdych** [comment]
If you don't mind I would like to add Notifier and AsyncNotifier for the "old" syntax. I was focus on the code generator syntax only however it would cover all riverpod 2.0 changes. I can do it tonight.

**Comment by trejdych** [comment]
Hello, I've fixed some indentations for code generator syntax and added few more snippets.

File: snippets/code_generator.code-snippets [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+			"\treturn $3;",
+			"}",
+		]
+	},
+	"Riverpod: simple keep alive instance": {
+        "scope": "dart",
+		"prefix": "riverpodKeepAlive",
+		"description": "Creates a simple keep alive Provider",
+		"body": [
+			"@Riverpod(keepAlive: true)",
```
Comment (leehack) [inline snippets/code_generator.code-snippets:26]: Typo?
`@Rivderpod` -> `@riverpod`

**Review by RobertBrunhage — APPROVED** [review]

**Comment by RobertBrunhage** [comment]
Sorry forgot this one, Looks good and will merge!

**Comment by trejdych** [comment]
Thank you @RobertBrunhage

**Comment by RobertBrunhage** [comment]
@trejdych seems like some of the new snippets wasn't added in the `package.json` if you get some time, would love to have a new PR with that!

**Comment by trejdych** [comment]
Ahh. Sorry. I'll fix it tonight

**Pull request merged**